webviz_config/containers/_table_plotter.py

```diff
@@ -188,12 +248,22 @@ def style_options_div_hidden(self):
     @property
     def style_page_layout(self):
         '''Simple grid layout for the page'''
-        return {} if self.lock else {
-            'display': 'grid',
-            'align-content': 'space-around',
-            'justify-content': 'space-between',
-            'grid-template-columns': '1fr 5fr'
-        }
+        if self.lock:
+            return {}
+        if self.use_filter:
+            return {
+                'display': 'grid',
+                'align-content': 'space-around',
+                'justify-content': 'space-between',
+                'grid-template-columns': '1fr 5fr 1fr'
+            }
+        else:
+            return {
+                'display': 'grid',
+                'align-content': 'space-around',
+                'justify-content': 'space-between',
+                'grid-template-columns': '1fr 5fr'
+            }
```

Comment:
These two are quite equal. Change to

1) Define dictionary outside if/else, except grid-template-columns.
2) Define grid-template-columns value using inline if-else

perhaps.